t/query/country_cities.rb

```diff
@@ -29,4 +29,18 @@
       end
     end
   end
+
+  describe 'United States of America (Q30)' do
+    around { |test| VCR.use_cassette('CountryCities Q30', &test) }
+
+    let(:cities) { Query::CountryCities.new(id: 'Q30').data }
+
+    describe 'Los Angeles (Q65)' do
+      subject { cities.find { |c| c.id == 'Q65' } }
+
+      it 'should have two legislatures' do
```

Comment:
It's worth noting that because of the changes in this PR, we do expect there to be two legislatures (i.e. this test is correct and useful). However there is a separate bug, #20, that says there actually should not be two legislatures (i.e. this test is wrong in the context of that bug).

I've included this test because it tests this change in behaviour, ignoring that bug. When that bug is subsequently fixed we can update this test too.